### ISSUE TITLE
LDAP auth doesn’t work immediately after start

### ISSUE THREAD

**Issue by cnotin** [flesueur/mi-lxc#41]
I noticed that if I try to use LDAP auth (eg with « mi-lxc.py attach commercial@target-commercial ») just a few seconds after starting the infra it fails. I have to wait a few couple seconds more to have it working...
I suppose the ldap server is still working on something and not operation yet.
Not a very big issue but quite surprising when we aren’t aware of it!

**Comment by flesueur** [comment]
Yes. It does not happen in normal usage since LDAP has largely enough time to start, but globally you're right. Even more, the global routing (with BGP) needs a few seconds (around 10 ?) to stabilize at edges. I think it is ok and anyway, I don't have an easy fix... Containers are started in parallel and detached at the very beginning of their boot process.

I could try to wait on has_ip after having started all of them, this may yield some partial improvement on this...

**Comment by cnotin** [comment]
Definitely not an issue for normal students :)

I'm wondering if the "attach" command could catch this specific error and add another message explaining to be more patient?

**Comment by flesueur** [comment]
Not activated, it will have to be parallelized (each check_ip seems to take 1-2 seconds for already ready containers)

**Comment by flesueur** [comment]
catching such a specific case with attach risks to be a bit hackish, I think. Well, let's keep that open and see if sth comes to mind at some point !
